.github/labeler.yml

```diff
@@ -1,34 +1,42 @@
 "review needed :eyes:":
   - src/**/*
+  - public/*
 
 "documentation :book:":
   - README.md
 
 "tooling :wrench:":
   - .github/**/*
   - src/scripts/*
+  - src/lib/*
+  - src/hooks/*
```

Comment:
Helper functions contained in these folders, so added them to tooling category. Can remove if we'd prefer to update separately:
```suggestion
```